src/components/ViewItem/ViewItem.js

```diff
@@ -136,8 +137,8 @@ class ViewItem extends React.Component {
       'barcode': loan => (<div data-test-item-barcode>{_.get(loan, ['item', 'barcode'])}</div>),
       'dueDate': loan => {
         return (
-          <div data-test-item-due-date>
-            <FormattedDate value={loan.dueDate} />
+          <div data-test-item-due-date className={css.loanDueDate}>
```

Comment:
@Dmitriy-Litvinenko , I agree, that does look nice, but we aim  to use the globally defined rules in [eslint-config-stripes](https://github.com/folio-org/eslint-config-stripes/blob/21e93e50874dd9a1912fc8c7d3ddb83a526d9f6f/index.js) to (a) enforce style consistently across repos and (b) keep the commentary on PRs focused on their content rather than their format. 

If you think the “split props onto separate lines when there is more than one” rule is worth enforcing, please open a PR against eslint-config-stripes and make a case for it at #stripes-architecture. Otherwise, I think we should let this personal preference slide.

Reply:
I agree. Let the robots complain about these kinds of things :)